src/ec/packages/ec-component-file/docs/file.md

```diff
@@ -0,0 +1,73 @@
+# ECL Twig - EC File component
+
+npm package: `@ecl-twig/ec-component-file`
+
+```shell
+npm install --save @ecl-twig/ec-component-file
+```
+
+## Parameters
+
+- "icon" (object) (default: {}): object of type Icon; file type
+- "title" (string) (default: '')
+- "language" (string) (default: '')
+- "meta" (string) (default: '')
+- "download" (object) (default: {}): object of type Link
+- "translation" (array) (default: []):
+  - "toggle" (object) (default: {}): object of type Button
+  - "items" (array) (default: []):
+    - "title" (string) (default: '')
+    - "meta" (string) (default: '')
+    - "lang" (string) (default: '')
+  - "description (string) (default:'')
+- "extra_classes" (optional) (string) (default: '') Extra classes (space separated) for the icon
+- "extra_attributes" (optional) (array) (default: []) Extra attributes for icon
```

Comment:
You can leave it until `Extra classes` or `attributes`, as it's easy to make it wrong with the specifics, i.e. `Extra attributes for icon`